config/maintenance/maintenance.go

```diff
@@ -12,6 +12,7 @@ var (
 	errInvalidMaintenanceStartFormat = errors.New("invalid maintenance start format: must be hh:mm, between 00:00 and 23:59 inclusively (e.g. 23:00)")
 	errInvalidMaintenanceDuration    = errors.New("invalid maintenance duration: must be bigger than 0 (e.g. 30m)")
 	errInvalidDayName                = fmt.Errorf("invalid value specified for 'on'. supported values are %s", longDayNames)
+	errInvalidTimezone               = errors.New("invalid timezone specified or format not supported. Use IANA timezone format (e.g. America/Sao_Paulo)")
```

Comment:
Can you update the README to include information about the timezone format to use?

Reply:
> Can you update the README to include this information about the timezone format to use?

Certainly, I've done it, but it seems a little awkward when I look at the line in the table with the parameter maintenance.timezone. I am referring to the length of the line. Is it okay, or should I change it?"

Reply:
Don't worry about it!